site/components/src/main/java/uk/nhs/digital/intranet/beans/NewsInternal.java

```diff
@@ -42,4 +52,26 @@ public LeadImageSection getLeadImageSection() {
     public String[] getKeys() {
         return getProperty("hippotaxonomy:keys");
     }
+
+    public List<NewsInternal> getLatestNews() throws QueryException {
+        HippoBean folder = RequestContextProvider.get().getSiteContentBaseBean();
+
+        HippoBeanIterator hippoBeans = HstQueryBuilder.create(folder)
+            .ofTypes(NewsInternal.class)
+            .where(and(
+                constraint("jcr:uuid").notEqualTo(this.getIdentifier()),
+                or(
+                    constraint("intranet:typeofnews").equalTo("permanent"),
+                    constraint("intranet:expirydate")
+                        .greaterOrEqualThan(Calendar.getInstance(), Resolution.HOUR)
+                ))
+            )
+            .orderByDescending("intranet:publicationdate")
+            .limit(5)
```

Comment:
Is there a chance they might want to change this value? If so, have you considered retrieving it from the component properties?

Reply:
I've kept it the same as the news on main website. If we want to make it configurable I will need to add it on the component instead of the bean and will need more ui changes. As far as I know it's never been changed on the main site

Reply:
fair enough